spec/lib/reporting_spec.rb

```diff
@@ -14,6 +14,10 @@
     create(:user, :registered, email: 'test2@example.com')
   end
 
+  let(:closed_user) do
```

Comment:
Rather than define another user, pick one and use the existing mechanism to close and redact it.